### PR TITLE
Add UI/E2E tests for conferences

### PR DESCRIPTION
**Pull request by cobbal**
refs: MBL-14189
affects: none
release note: none

[run-nightly]

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/d4wteh7) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fd4wteh7)
teacher: [Link](https://rebrand.ly/h8pf40e) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fh8pf40e)
parent: [Link](https://rebrand.ly/j0ckrr0) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fj0ckrr0)

File: Student/StudentE2ETests/Conferences/ConferencesE2ETests.swift [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        inProgress.title.tap()
+
+        ConferenceDetails.join.waitToExist()
+        XCTAssertEqual(ConferenceDetails.title.label(), "The Eternal Conference")
+        XCTAssert(ConferenceDetails.status.label().contains("In Progress | Started Apr 8, 2020 at "))
+        XCTAssertEqual(ConferenceDetails.details.label(), "It never ends...")
+    }
+
+    func test() {
+        RunLoop.current.run()
```
Comment (armstrongnate) [inline Student/StudentE2ETests/Conferences/ConferencesE2ETests.swift:66]: What is this about?

Comment (cobbal) [reply Student/StudentE2ETests/Conferences/ConferencesE2ETests.swift:66]: Oops... That was just for playing around. It hangs forever.

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

[MBL-14189](https://instructure.atlassian.net/browse/MBL-14189)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.6% | 94.14% | -0.53%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**